includes/publisher/class-publisher-form-editing-experience.php

```diff
@@ -403,7 +403,7 @@ public function should_show_feedback_banner(): bool {
 	 * Runs an `admin_notices` hook.
 	 */
 	public function render_feedback_notice(): void {
-		include_once ATLAS_CONTENT_MODELER_DIR . '/includes/shared-assets/views/banners/atlas-content-modeler-feedback-banner.php';
+		include_once ATLAS_CONTENT_MODELER_INCLUDES_DIR . '/includes/shared-assets/views/banners/atlas-content-modeler-feedback-banner.php';
```

Comment:
I think we need to drop the `/includes/` here if `ATLAS_CONTENT_MODELER_INCLUDES_DIR` is going to contain `/includes/`?

```suggestion
		include_once ATLAS_CONTENT_MODELER_INCLUDES_DIR . 'shared-assets/views/banners/atlas-content-modeler-feedback-banner.php';
```